SPECS/k3s-calico/k3s-calico.spec

```diff
@@ -1,22 +1,26 @@
 Name:           k3s-calico
 Version:        3.30.1
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        Calico manifests and container images for k3s kubernetes cluster.
 
 License:        Apache-2.0
 Vendor:         Intel Corporation
 Distribution:   Edge Microvisor Toolkit
 
 URL:            https://github.com/projectcalico/calico
-Source0:        https://github.com/projectcalico/calico/releases/download/v%{version}/release-v%{version}.tgz#/%{name}-%{version}.tar.gz
+Source0:        https://github.com/projectcalico/calico/archive/refs/tags/v%{version}.tar.gz#/%{name}-v%{version}.tar.gz
+# Generate tar.zst images by running `docker save <image> | zstd -T0 -16 -f --long=25 > <image>.tar.zst`
+Source1:        %{name}-cni-image-v%{version}.tar.zst
+Source2:        %{name}-node-image-v%{version}.tar.zst
+Source3:        %{name}-controllers-image-v%{version}.tar.zst
```

Comment:
We can use the image tars from Source0 to create compressed image sources? That would be less manual job future upgrades. Though, I'm okay with this for now.

Reply:
We went this route to keep consistent with the other specs and to save build times (though that part doesn't really matter) - we can revert and compress during package build in a future change if needed.